subprojects/common/junit-utils/src/main/java/com/avito/test/Flaky.kt

```diff
@@ -5,6 +5,7 @@ package com.avito.test
  * It's used for retry logic.
  */
 @Retention(AnnotationRetention.BINARY)
+@Target(AnnotationTarget.CLASS) // member functions are not supported by 'org.gradle.test-retry' plugin
```

Comment:
Do we have a link to the issue?

Reply:
I suppose it's expected behavior for the `includeAnnotation<Classes>` property :)

Reply:
In this case, the comment about supporting in `test-retry` plugin is confusing. It looks like referring to a bug in the plugin.